cli/lsp/documents.rs

```diff
@@ -350,15 +324,15 @@ impl DocumentCache {
 
   pub fn line_index(&self, specifier: &ModuleSpecifier) -> Option<LineIndex> {
     let doc = self.docs.get(specifier)?;
-    doc.line_index.clone()
+    Some(doc.source().line_index().clone())
```

Comment:
This could be changed to not clone, but we can worry about that in a future refactoring...